modules/virt-configuring-vm-disk-sharing.adoc

```diff
@@ -10,6 +10,14 @@ You can configure block volumes so that multiple virtual machines (VMs) can shar
 
 The application running on the guest operating system determines the storage option you must configure for the VM. A disk of type `disk` exposes the volume as an ordinary disk to the VM.
 
+You can set an error policy for each disk. The error policy controls how the hypervisor behaves when an input/output error occurs when a disk is being written to or read. The default behavior stops the guest and generates a Kubernetes event.
```

Comment:
Get rid of the two when's:
```suggestion
You can set an error policy for each disk. The error policy controls how the hypervisor behaves when an input/output error occurs while a disk is being written to or read. The default behavior stops the guest and generates a Kubernetes event.
```

Reply:
> stops the guest

Will a typical user know what a _guest_ is here?

Reply:
@alicefr , do you feel that we need to expand upon what is meant by _guest_ in the doc update?

Reply:
I'm maybe not the right person to answer this question since for me as a developer, guest is a synonym of virtual machine. We also use as persona the "guest owner" in order to identify the user which is using the virtual machine. IMO, guest is a clear concept, but it could be replaced by VM without problems

Reply:
Thanks, @alicefr . I was thinking the same thing and will make the change from 'guest' to 'VM.'